src/input/index.js

```diff
@@ -291,6 +293,7 @@ class TonicInput extends Tonic { /* global Tonic */
     const valueAttr = value && value !== 'undefined' ? `value="${value}"` : ''
 
     const attributes = [
+      idAttr,
```

Comment:
This may cause a problem if the ID is passed to the label as well we will end up with two elements with the same id. Is there a 508 rule that says it must be on the label?

Reply:
ah I see that the inner element's id is prefixed. :+1: